src/components/date-picker/DatePicker.js

```diff
@@ -383,7 +383,7 @@ class DatePicker extends React.Component<Props> {
         const { formatMessage } = intl;
         const classes = classNames(className, 'date-picker-wrapper', {
             'show-clear-btn': !!value,
-            'show-error': !!error,
+            'bdl-show-error': !!error,
```

Comment:
I think SUIT states are meant to begin with `is-`, though maybe we can include `has-`. Should this be `bdl-is-error` or `bdl-has-error`? Or should it be a modifier, such as `bdl-DatePicker--invalid`?

Reply:
This is used in a bunch of places, so I will change it to `bdl-has-error`